GNUmakefile.llvm

```diff
@@ -300,8 +309,8 @@ endif
 ifneq "$(LLVM_CONFIG)" ""
   CLANG_CFL += -I$(shell dirname $(LLVM_CONFIG))/../include
 endif
-CLANG_CPPFL  = `$(LLVM_CONFIG) --cxxflags` -fno-rtti -fno-exceptions -fPIC $(CXXFLAGS) $(CPPFLAGS) -Wno-deprecated-declarations
-CLANG_LFL    = `$(LLVM_CONFIG) --ldflags` $(LDFLAGS)
+CLANG_CPPFL  = $$($(LLVM_CONFIG) --cxxflags) -fno-rtti -fno-exceptions -fPIC $(CXXFLAGS) $(CPPFLAGS) -Wno-deprecated-declarations
```

Comment:
Ah seems to be some syntax I don't know :D

Reply:
How does $$ work?

Reply:
That’s why I don’t want to touch it ;)

Reply:
`$$` is simply escaping the `$` at GNU Make level. I.e. it's passing the `$` to the shell. POSIX shells have supported this since the POSIX 1993 standard, so one would have to attempt to build this ins a rather ancient setup (which in turn is at odds with the requirements set out to build AFL++).